prow/config.yaml

```diff
@@ -227,6 +227,7 @@ tide:
     - kubernetes/website
     - kubernetes-sigs/application
     - kubernetes-sigs/cluster-api
+    - kubernetes-sigs/cluster-api-provider-gcp
```

Comment:
before we enable merge automation we also need to at least enable in `prow/plugins.yaml`: `approve` for this repo so `/approve` works, this merge query requires certain labels (see line 239 here)

Reply:
I've added the config to `prow/plugins.yaml` and squashed it into this PR.